src/Service.Tests/Configuration/ConfigurationTests.cs

```diff
@@ -514,37 +515,80 @@ public void CleanupAfterEachTest()
         }
 
         /// <summary>
-        /// Checks if the connection string provided in the config is correctly updated for MSSQL.
+        /// Checks if the connection string provided in the config is correctly updated for MSSQL by checking for the injected User Agent string w/ format:
```

Comment:
nit: some of these lines in the summary comment are a bit long and would benefit from being broken up.